### PR TITLE
wg: run discourse on tidb

### PR DESCRIPTION
**Pull request by sykp241095**
Add working group for: run discourse on TiDB.

### CODE REVIEW

File: working-groups/wg-run-discourse-on-tidb.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Run Discourse on TiDB Working Group
+
+## Goals
+
+Run Discourse on TiDB instead of PostgreSQL
+
+## Meetings
+
```
Comment (winkyao) [inline working-groups/wg-run-discourse-on-tidb.md]: Please make sure your working group meeting time, and the meeting notes link.

File: working-groups/wg-run-discourse-on-tidb.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+## Members
+
+* [hooopo](https://github.com/hooopo)
+* [JinLingChristopher](https://github.com/JinLingChristopher)
+
+## Contact
+
+* Slack: channel **#wg-run-discourse-on-tidb** in the
+  [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```
Comment (gingerkidney) [inline working-groups/wg-run-discourse-on-tidb.md]:
```suggestion
  [tidbcommunity](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) slack workspace.
```

Comment (gingerkidney) [reply working-groups/wg-run-discourse-on-tidb.md]: Please create a slack channel

File: working-groups/wg-run-discourse-on-tidb.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Members
+
+* [wd0517](https://github.com/wd0517), [PingCAP](https://github.com/pingcap)
+* [JinLingChristopher](https://github.com/JinLingChristopher)
+
+## Contact
+
+* Slack: channel **#wg-run-discourse-on-tidb** in the
+  [tidbcommunity](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE) slack workspace.
```
Comment (zz-jason) [inline working-groups/wg-run-discourse-on-tidb.md]: slack url: https://pingcap.com/tidbslack/

File: working-groups/wg-run-discourse-on-tidb.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Run Discourse on TiDB Working Group
+
+## Goals
+
+Run Discourse on TiDB instead of PostgreSQL
+
+## Meetings
+
+* Regular Working Group Meeting: [UTC+8]2:30pm on every Thursday
```
Comment (zz-jason) [inline working-groups/wg-run-discourse-on-tidb.md]: It's better to add a url to convert this time to other time zones.

File: working-groups/wg-run-discourse-on-tidb.md (hunk 3 above)
Comment (gingerkidney) [reply working-groups/wg-run-discourse-on-tidb.md]: this is not an invitation link

**Comment by sykp241095** [comment]
> please update `working-groups/README.md`, list this working group to the active working groups section.

Added

**Pull request merged**